src/accountsdb/buffer_pool.zig

```diff
@@ -735,190 +757,356 @@ pub const HierarchicalFIFO = struct {
 
 /// slice-like datatype
 /// view over one or more buffers owned by the BufferPool
-pub const CachedRead = struct {
-    buffer_pool: *BufferPool,
-    frame_indices: []const FrameIndex,
-    /// inclusive, the offset into the first frame
-    first_frame_start_offset: FrameOffset,
-    /// exclusive, the offset into the last frame
-    last_frame_end_offset: FrameOffset,
+pub const ReadHandle = union(enum) {
```

Comment:
AccountDataHandle ?

Reply:
Renamed to AccountDataHandle f40689214b7f8a0945d24c5a2847c29af6db0203

Reply:
still some places like `const ReadHandle = sig.accounts_db.buffer_pool.AccountDataHandle;`

Reply:
Hm I think I got all of them, `grep -r "ReadHandle" src/` shows nothing.
```
src/accountsdb/accounts_file.zig:
  11: const AccountDataHandle = sig.accounts_db.buffer_pool.AccountDataHandle;
src/accountsdb/cache.zig:
  12: const AccountDataHandle = sig.accounts_db.buffer_pool.AccountDataHandle;
src/accountsdb/db.zig:
  37: const AccountDataHandle = sig.accounts_db.buffer_pool.AccountDataHandle;
src/accountsdb/fuzz.zig:
  14: const AccountDataHandle = sig.accounts_db.buffer_pool.AccountDataHandle;
src/core/account.zig:
  6: const AccountDataHandle = sig.accounts_db.buffer_pool.AccountDataHandle;
  ```